pkg/proxy/request.go

```diff
@@ -140,7 +143,19 @@ func fromHTTPRequestToLoadTestSpec(r *http.Request, logger *zap.Logger) (apisLoa
 		return apisLoadTestV1.LoadTestSpec{}, fmt.Errorf("error getting %q from request: %w", duration, err)
 	}
 
-	return backends.BuildLoadTestSpecByBackend(ltType, o, dp, tagList, tf, td, ev, turl, dur)
+	masterImageRef, err := getMasterImageRef(r)
```

Comment:
This should be defined in environment variables(https://github.com/hellofresh/kangal/blob/master/pkg/backends/locust/config.go#L5-L6), and not in HTTP request.

Reply:
Then I believe the environment variable must be removed; this config only exists in the controller side.
We established that we must not update the Spec from the controller side, so it must be configured from the proxy.

Reply:
Since the proxy is responsible for creating load test specs those env vars should be also accessible there

Reply:
But I believe we should not expose it on HTTP request

Reply:
For me, either the proxy should expose this or remove those properties from the Spec. Otherwise the only way to use this feature is by manually creating LoadTest resources, encouraging users to not use Proxy.

Reply:
> Since the proxy is responsible for creating load test specs those env vars should be also accessible there

This is now done via 2aa6dd5 829ac60; proxy will now use the same environment variables and use it when creating new loadtests.

>> But I believe we should not expose it on HTTP request

> For me, either the proxy should expose this or remove those properties from the Spec. Otherwise the only way to use this feature is by manually creating LoadTest resources, encouraging users to not use Proxy.

I agree with @diegomarangoni; I think we should either use it (i.e. expose it via the proxy) or remove this from the spec as it has no function.